dear_petition/petition/export/mapper.py

```diff
@@ -18,12 +19,13 @@ def build_pdf_template_context(petition, extra):
 
 
 def map_petition(data, petition, extra={}):
-    data["County"] = petition.county
-    if petition.jurisdiction == constants.DISTRICT_COURT:
+    data["County"] = getattr(petition, "county", None)
```

Comment:
The model requires these fields, so they should always be set. I think we can just use the ``data["County"] = petition.county`` format if possible.